test/controllers/update/check.your.answers.controller.spec.ts

```diff
@@ -246,6 +247,7 @@ describe("CHECK YOUR ANSWERS controller", () => {
       expect(resp.text).toContain(UPDATE_CHANGE_LINK_REVIEWED_BO_INDIVIDUAL);
       expect(resp.text).toContain(UPDATE_CHANGE_LINK_REVIEWED_BO_GOVERNMENT);
       expect(resp.text).toContain(UPDATE_CHANGE_LINK_REVIEWED_BO_OTHER);
+      expect(resp.text).toContain(HOME_ADDRESS_LINE1);
```

Comment:
I see "addressLine1" in several addresses - are we sure we are getting the one from private one?